docs/competition/teams/role-descriptions/intro-briefer.md

```diff
@@ -0,0 +1,38 @@
+---
+original:
+  authors: Jeremy Morse, Rob Gilton and Sean Bedford
+  url: https://github.com/srobo/comp-team-coord-docs/RoleDescriptions/intro-briefer
+  extra: |
+    The original work is based on work done by Jeremy Morse, which was under the
+    Creative Commons Attribution 4.0 License.  To view a copy of this license,
+    visit <a href="https://creativecommons.org/licenses/by/4.0/">https://creativecommons.org/licenses/by/4.0/</a>.
+---
+# Introductory Briefer
+
+The introductory and H&S briefer gives a ~10m talk at the beginning of the
+competition regarding matters of importance, which at a high level are:
+
+ * Health and Safety
+ * How the competition operates
+
+In that order.
+
+Health & Safety should cover at least:
+
+ * Common sense (i.e., don't run everywhere)
+ * Use of tools
+ * Battery safety
+ * Robot safety checking
+
+The competition intro talk should:
+
+ * Introduce tinker time
+ * Explain (and illustrate) the procedure for operating the robot in a match,
+   from gate to gate.
+   * Especially highlight that teams must insert the USB stick provided at the staging area  **before** turning on their robot at the beginning of the match.
+ * Lay down rules on who is allowed where
+ * Describe tinker time and make it clear how one books tinker time.
+ * Give a short overview of the game itself (more or less what is printed on the competition flyer). Some important points to highlight:
```

Comment:
We can remove the game-specific parts of this

Reply:
c76d4ff